### DISCUSSION TITLE
New color palette for the GUI

### DISCUSSION THREAD

**Discussion by mario-nt** [torrust/torrust-index-gui#429] [Proposals]
Regarding the addition of the future light/dark themes for the GUI, I have come with the idea of using the same color palette of the Torrust website.

It would give both the GUI and the website a cohesive, professional and easily recognizable look.

I want to know what each member of the team think about this.

Also maybe it would be a good idea to ask Matt Fantinel what he thinks about it, as he is the author of the theme.

@josecelano @da2ce7 @WarmBeer @cgbosse @grmbyrn

**Comment by josecelano** [comment]
Hi @mario-nt I think it's a complex topic. There are still a lot of open questions like:

- Whether we support custom themes for the Index in the future or not.
- What level of customization do we want for the Index (only basic stuff like logo, colour, ... or layout, font, ...)
- When are we going to implement features related to UX/UI. We haven't defined a roadmap. Is UX/UI a priority right now or can we continue using the current interface?
- Should we use our storybook? or should we migrate to a component library?
- Are we going to continue using NUXT/VUE or could we migrate in the future to Svelte?

Some of them are implementation details that should not be affected by this decision but I think it would be better to answer them, especially if the changes needed take long.

But In general, I do not see the need to keep both the application and the organization design similar.

**Reply by mario-nt** [reply]
I agree with what you pointed out. Things like the framework that we will be using in the future should be decided beforehand. 
I think for the time being, we can improve the GUI a bit, and then, later on, consider continue working on the rest of the features.
